package.json

```diff
@@ -17,8 +17,9 @@
     "colorfont"
   ],
   "scripts": {
-    "test": "node_modules/.bin/mocha --reporter mochawesome --reporter-options reportDir=test/report,reportFilename=report,json=false,code=false,cdn=true,reportTitle=OpenMoji-Tester,reportPageTitle=OpenMoji-Tester test/*.js --openmoji-data-json $PWD/data/openmoji.json --openmoji-src-folder $PWD/src",
-    "test-dev": "node_modules/.bin/mocha --invert --grep \"production\" test/*.js --openmoji-data-json $PWD/data/openmoji.json --openmoji-src-folder $PWD/src",
```

Comment:
The relevant changes here are the removal of `node_modules/.bin` since `npm` ensures this is on the `PATH` and the removal of `$PWD/` since this is also ensured to be the project root by npm.

Reply:
Odd. Had to put the PWDs back. Must be something odd about mocha specifically.